assets/sass/components/idea-hub/_googlesitekit-idea-hub-common.scss

```diff
@@ -0,0 +1,24 @@
+/**
+ * Idea Hub common styles.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-idea-hub__badge {
+		margin-left: 12px;
+	}
```

Comment:
@aaemnnosttv, unfortunately, it doesn't work well on small devices. I believe `padding-right: 12px` was added on purpose to make sure that the badge won't have an empty margin on the left when it gets wrapped to the next line. See how `develop` looks on small mobiles:
![2021-09-23_20-51](https://user-images.githubusercontent.com/873049/134558513-ef4e8e55-3aa4-43c7-b051-026d70f5a6ce.png)

And how your branch:
![2021-09-23_20-50](https://user-images.githubusercontent.com/873049/134558413-3e6802b4-2cbd-4f6d-a15a-811b68da90e5.png)

Reply:
Good catch! I was wondering why it was done that way 😄  I've updated this one instance to keep the same styles as before without the added class on the badge as an exception but with an added comment to explain.